src/av2/structures/sweep.py

```diff
@@ -43,7 +45,7 @@ class Sweep:
     ego_SE3_down_lidar: SE3
 
     def __len__(self) -> int:
-        """Return the number of LiDAR returns in the aggregated sweep."""
+        """Return the number of LiDAR returns in the sweep."""
```

Comment:
```suggestion
        """Return the number of lidar returns in the sweep."""
```

Nit: We used "lidar" in the paper instead of "LiDAR" (just for consistency).